CONTRIBUTING.md

```diff
@@ -1,3 +1,227 @@
+  Slack channel or have particular interest to the community as a whole. Depending on the
+  discussion, these can turn into `feature` or `bug` issues.
+- `proposal`: Used for items (like this one) that propose a new ideas or functionality that require
+  a larger community discussion. This allows for feedback from others in the community before a
+  feature is actually  developed. This is not needed for small additions. Final word on whether or
+  not a feature needs a proposal is up to the core maintainers. All issues that are proposals should
+  both have a label and an issue title of "Proposal: [the rest of the title]." A proposal can become
+  a `feature` and does not require a milestone.
+- `feature`: These track specific feature requests and ideas until they are complete. They can
+  evolve from a `proposal` or can be submitted individually depending on the size.
+- `bug`: These track bugs with the code
+- `docs`: These track problems with the documentation (i.e. missing or incomplete)
+
+### Issue Lifecycle
+
+The issue lifecycle is mainly driven by the core maintainers, but is good information for those
+contributing to Helm. All issue types follow the same general lifecycle. Differences are noted
+below.
+
+1. Issue creation
+2. Triage
+    - The maintainer in charge of triaging will apply the proper labels for the issue. This includes
+      labels for priority, type, and metadata (such as `good first issue`). The only issue priority
+      we will be tracking is whether or not the issue is "critical." If additional levels are needed
+      in the future, we will add them.
+    - (If needed) Clean up the title to succinctly and clearly state the issue. Also ensure that
+      proposals are prefaced with "Proposal: [the rest of the title]".
+    - Add the issue to the correct milestone. If any questions come up, don't worry about adding the
+      issue to a milestone until the questions are answered.
+    - We attempt to do this process at least once per work day.
+3. Discussion
+    - Issues that are labeled `feature` or `proposal` must write a Helm Improvement Proposal (HIP).
+    - Issues that are labeled as `feature` or `bug` should be connected to the PR that resolves it.
+    - Whoever is working on a `feature` or `bug` issue (whether a maintainer or someone from the
+      community), should either assign the issue to themselves or make a comment in the issue saying
+      that they are taking it.
+    - `proposal` and `support/question` issues should stay open until resolved or if they have not
+      been active for more than 30 days. This will help keep the issue queue to a manageable size
+      and reduce noise. Should the issue need to stay open, the `keep open` label can be added.
+4. Issue closure
+
+## How to Contribute a Patch
+
+1. Identify or create the related issue. If you're proposing a larger change to
+   Helm, see [Proposing an Idea](#proposing-an-idea).
```

Comment:
There is no section on "Proposing an Idea". I think you can just remove "1." and only have a 2 item list.

Reply:
done